src/index.js

```diff
@@ -0,0 +1,83 @@
+import TodoList from './todoList';
+import Todo from './todo';
+import Projects from './projects';
+import Project from './project';
+
+const projects = new Projects();
+projects.listProjects();
+const todoList = new TodoList();
+todoList.listTodos();
+
+function clearFields() {
```

Comment:
## Required change 
#### You need to make sure you are using ES6 syntax as described in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/)

- [ ] In this instance, you ought to have employed the usage of an arrow function instead of this anonymous `function` expression.